dask/dataframe/io/parquet/arrow.py

```diff
@@ -106,6 +107,7 @@ def _write_partitioned(
             subschema = subschema.remove(subschema.get_field_index(col))
 
     md_list = []
+    metadata_collector = {"metadata_collector": md_list} if return_metadata else {}
```

Comment:
I find this pattern (defining a kwargs dict instead of a single parameter) a bit hard to read - would `metadata_collector=None` not work when passed to `write_table`?

Reply:
Ahh - Right. That is **much** better. Setting `None` should be the default :)